src/hotspot/share/opto/machnode.cpp

```diff
@@ -23,6 +23,7 @@
  */
 
 #include "precompiled.hpp"
+#include "gc/shared/barrierSet.hpp"
```

Comment:
Seems to need `#include "gc/shared/c2/barrierSetC2.hpp"` on some platforms (just looking at your GHA)

Reply:
Good catch, done!